pkg/nodebootstrap/assets/bootstrap.al2.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -eu
 
-echo "NODE_IP=$(hostname -i)" > /etc/eksctl/kubelet.local.env
+echo "NODE_IP=$(curl 169.254.169.254/latest/meta-data/local-ipv4)" > /etc/eksctl/kubelet.local.env
```

Comment:
Very nit but I prefer having the explicit `http://` scheme in front of the metadata service ip, just because I don't know much about how `curl` in every distribution works when the scheme is missing :)

Reply:
Please consider this comment as already addressed by #633bd25. Thanks for your effort @mukaibot!